apps/src/code-studio/components/progress/ScriptOverviewTopRow.jsx

```diff
@@ -87,6 +95,19 @@ const ScriptOverviewTopRow = React.createClass({
             assignmentName={scriptTitle}
           />
         )}
+        {!professionalLearningCourse && viewAs === ViewType.Teacher &&
+            resources.length > 0 &&
+          <div style={styles.dropdown}>
+            <DropdownButton
+              text={i18n.teacherResources()}
+              color={Button.ButtonColor.blue}
+            >
+              {resources.map(({type, link}, index) =>
+                <a key={index} href={link}>{stringForType[type]}</a>
```

Comment:
The challenge is that sometimes the element has an href and sometimes it has an onclick. Instead of passing an array of children, I _could_ pass an array of tuples, where each is a link/onClick/string. Just passing in anchor tags directly as children seemed more clear to me though.